cts_can/cts_can.py

```diff
@@ -118,6 +120,22 @@ def command(
         verbose=False):
     # type: (object, object, object, object, object, object, object, object, object) -> object
 
+    msgID = None
+    if cmdtype == 'Abort':
+        msgID = 0x00
+    elif cmdtype == 'SetCANAddress':
+        msgID = 0x01
+    elif cmdtype == 'SetDACLevel':
+        msgID = 0x02
+    elif cmdtype == 'SetLED':
+        msgID = 0x03
+    elif cmdtype == 'GetLEDandDAC':
+        msgID = 0x04
+    elif cmdtype == 'GetVersion':
+        msgID = 0x1E
+    else:
+        raise Exception('Invalid command type' + cmdtype)
+
```

Comment:
looks like `cmdtype` might want to be an `enum.Enum` or `enum.IntEnum` rather than a string...

Reply:
Alternatively one could use a conversion-table like this:
```python:
table = {
    'Abort': 0x00,
    'SetCANAddress': 0x01,
    'SetDACLevel': 0x02,
    'SetLED': 0x03,
    'GetLEDandDAC': 0x04,
    'GetVersion': 0x1E,
}
msgID = table[cmdtype]
```
This raises "KeyError" automatically for you, when there is nothing in the table, which fits to `cmdtype`.